docs/documentation/usage.rst

```diff
@@ -148,46 +160,60 @@ Problem definition
 
 .. code:: yaml
 
-      # Although "model" is a mandatory name for the top level of the model, its
-      # sub-components can be freely named by user
-      geometry:
-        # An OpenMDAO component is identified by its "id"
-        id: fastoad.geometry.legacy
-      weight:
-        id: fastoad.weight.legacy
-      mtow:
-        id: fastoad.mass_performances.compute_MTOW
-      aerodynamics_highspeed:
-        id: fastoad.aerodynamics.highspeed.legacy
-      aerodynamics_lowspeed:
-        id: fastoad.aerodynamics.lowspeed.legacy
-      aerodynamics_takeoff:
-        id: fastoad.aerodynamics.takeoff.legacy
-      aerodynamics_landing:
-        id: fastoad.aerodynamics.landing.legacy
-        use_xfoil: no
+      # Components can be put in sub-groups
+      subgroup:
+
+        # A group can be set with its own solvers.
+        # Uncomment the 2 next lines for using the performance module with "mission_file_path: ::sizing_mission"
+        #nonlinear_solver: om.NonlinearBlockGS(maxiter=100, atol=1e-2, iprint=0)
+        #linear_solver: om.DirectSolver()
+
+        geometry:
+          # An OpenMDAO component is identified by its "id"
+          id: fastoad.geometry.legacy
+        weight:
+          id: fastoad.weight.legacy
+        mtow:
+          id: fastoad.mass_performances.compute_MTOW
+        hq_tail_sizing:
+          id: fastoad.handling_qualities.tail_sizing
+        hq_static_margin:
+          id: fastoad.handling_qualities.static_margin
+        wing_position:
+          id: fastoad.loop.wing_position
+        aerodynamics_highspeed:
+          id: fastoad.aerodynamics.highspeed.legacy
+        aerodynamics_lowspeed:
+          id: fastoad.aerodynamics.lowspeed.legacy
+        aerodynamics_takeoff:
+          id: fastoad.aerodynamics.takeoff.legacy
+        aerodynamics_landing:
+          id: fastoad.aerodynamics.landing.legacy
+          use_xfoil: false
       performance:
         id: fastoad.performances.mission
         propulsion_id: fastoad.wrapper.propulsion.rubber_engine
         mission_file_path: ::sizing_breguet
-        # mission_file_path: ::sizing_mission
-        out_file:  ./flight_points.csv
-        adjust_fuel: yes
-        is_sizing: yes
-      hq_tail_sizing:
-        id: fastoad.handling_qualities.tail_sizing
-      hq_static_margin:
-        id: fastoad.handling_qualities.static_margin
+        # mission_file_path: ::sizing_mission  # Activate solvers in "subgroup" if you use this line.
```

Comment:
```suggestion
        # mission_file_path: ::sizing_breguet
        mission_file_path: ::sizing_mission
```

Reply:
Thanks for this remark, you are right it makes more sense to uncomment the solvers so that their usage within a subgroup can be apprehended. Modifications were commited